eva/server/db_api.py

```diff
@@ -76,15 +77,20 @@ def _upload_transformation(self, query: str) -> str:
          the content into a string.
         """
         if 'UPLOAD' in query:
-            file_path = query.split()[2][1:-1]
-            dst_path = query.split()[-1][1:-2]
+            query_list = query.split()
```

Comment:
Please drop a comment here. We might forget what is happening here.

Reply:
Needs to be resolved.